README.md

```diff
@@ -57,6 +60,11 @@ This library takes a set of different options whose expanded map is provided bel
       }
     });
 ```
+- `browser`: NOT OPTIONAL. Set to `true` for regular in-browser usage. `false` is used for debug purposes (prints results in the console).
+- `caching`: Enables cache mechanism for repetitive point detections. Defaults to `true`.
+- `leniency`: Minimum threshold value for which a point qualifies as a "match". Defaults to `30`%.

```

Comment:
Maybe clarify that it is a percentage, or offer a range of OK values?

Reply:
Done here https://github.com/publiclab/matcher-core/blob/add-readme/README.md#arguments